src/main/kotlin/no/nav/syfo/AccessTokenClient.kt

```diff
@@ -19,31 +21,37 @@ class AccessTokenClient(
         private val client: HttpClient
 ) {
     private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.syfonarmesteleder")
+    private val mutex = Mutex()
+    @Volatile
+    private var token: AadAccessToken? = null
 
     suspend fun hentAccessToken(resource: String): String {
-        log.trace("Henter token fra Azure AD")
-        // TODO: Remove this workaround whenever ktor issue #1009 is fixed
-        val response: AadAccessToken = client.post<HttpResponse>(aadAccessTokenUrl) {
-            accept(ContentType.Application.Json)
-            method = HttpMethod.Post
-            body = FormDataContent(Parameters.build {
-                append("client_id", clientId)
-                append("resource", resource)
-                append("grant_type", "client_credentials")
-                append("client_secret", clientSecret)
-            })
-        }.use { it.call.response.receive<AadAccessToken>() }
-        log.trace("Har hentet accesstoken")
-        return response.access_token
+        val omToMinutter = Instant.now().plusSeconds(120L)
```

Comment:
Vi snakke om det tidligere: Denne er i UTC+0 ikke sant? Vi får ikke noe bøll med sommertid eller sånt fjas?

Reply:
En `Instant` er vel bare en epoch-teller uavhening av tidssoner og sommertid?
Regner med at det er det samme som er brukt i tokenet også, hvis det der også bare er ett stort tall for tid...

Reply:
Ikke sant. Det er jo nøyaktig det vi vil ha 😻

Reply:
Ja, det stemmer, det er det som brukes i tokenet :)
>The time when the access token expires. The date is represented as the number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time. This value is used to determine the lifetime of cached tokens.

https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-client-creds-grant-flow